cmd/albatross/albatross.go

```diff
@@ -44,8 +47,18 @@ func startServer() {
 	router.Handle("/install", ContentTypeMiddle(installHandler)).Methods(http.MethodPut)
 	router.Handle("/upgrade", ContentTypeMiddle(upgradeHandler)).Methods(http.MethodPost)
 
+	serveDocumentation(router)
 	err := http.ListenAndServe(fmt.Sprintf(":%d", 8080), router)
 	if err != nil {
 		logger.Errorf("error starting server", err)
 	}
 }
+
+func serveDocumentation(r *mux.Router) {
+	docEnv := os.Getenv("DOCUMENTATION")
```

Comment:
Maybe we can rename this to SERVE_DOCUMENTATION?

Reply:
Also do we need this conditional? Should we always expose the path for api docs?

Reply:
We could always expose the API as well.